frontend/shared/src/components/confirm-dialog/ConfirmDialog.tsx

```diff
@@ -22,6 +22,12 @@ const ConfirmDialog: React.FC<Partial<ModalProps>> = (modalProps) => {
       {...modalProps}
     >
       {confirmState.content?.length ? confirmState.content : null}
```

Comment:
Okay it seem to be set as method param, but it could be like:
```
await confirm({
      header: t('common:publish.confirmation', { posting: posting.title }),
      children: <>{t('common:application.publishTerms')}<a target="_blank" rel="noopener noreferrer" href="/TET-alusta-kayttoehdot.pdf">
          {' '}
          {t('common:application.termsLink')}
        </a></>
      
      ,
      linkText: t('common:application.termsLink'),
      link: '/TET-alusta-kayttoehdot.pdf',
      submitButtonLabel: t('common:publish.publishPosting'),
    });

```